travel-support.md

```diff
@@ -0,0 +1,17 @@
+---
+layout: default
+title: Travel Support
+slug: travel-support
+---
+
+## Travel Support
+
+The organisers are arranging a travel support fund for those who would otherwise be unable to attend sustain in San Francisco on 19th June.  Travel support will cover the costs of ground and air transportation, accomodation and meals along with the cost of a ticket. 
+
+Sustain OSS is all about supporting open source maintainers. We want to hear from maintainers with a deep understanding of the issues and from those who have tested ideas to improve their own projects. With that in mind the organisers will prioritise support for those with experience of maintainting or sustianing and open source project. 
```

Comment:
maintianting > maintaining. 
sustianing > sustaining.

While we're at it - maybe American spelling of `organizers`?